src/general/EditorBase.cs

```diff
@@ -789,26 +789,7 @@ private void ApplyAutoEvoResults()
     {
         var run = CurrentGame.GameWorld.GetAutoEvoRun();
         GD.Print("Applying auto-evo results. Auto-evo run took: ", run.RunDuration);
-        run.ApplyExternalEffects();
-
-        CurrentGame.GameWorld.Map.UpdateGlobalTimePeriod(CurrentGame.GameWorld.TotalPassedTime);
-
-        // Update populations before recording conditions - should not affect per-patch population
-        CurrentGame.GameWorld.Map.UpdateGlobalPopulations();
-
-        // Needs to be before the remove extinct species call, so that extinct species could still be stored
-        // for reference in patch history (e.g. displaying it as zero on the species population chart)
-        foreach (var entry in CurrentGame.GameWorld.Map.Patches)
-        {
-            entry.Value.RecordSnapshot(true);
-        }
-
-        var extinct = CurrentGame.GameWorld.Map.RemoveExtinctSpecies(FreeBuilding);
-
-        foreach (var species in extinct)
-        {
-            CurrentGame.GameWorld.RemoveSpecies(species);
-        }
+        run.ApplyAllEffects(FreeBuilding);
```

Comment:
I'd rename ApplyAllEffects to ApplyAllResultsAndEffects to make it clear that that is what does everything auto-evo needs to do after a run.